docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -65,6 +65,11 @@ The main features of the Media Controller API include:
 
   You can [set the media controller server features](#media-controller-server-features) using the server interfaces.
 
+- Sending custom events
+
+  You can [use the server to send events](#send_custom_events) with the `sendEvent()` method.
```

Comment:
Please make this small change:

You can [use the server to send the custom events](#send_custom_events) with the `sendEvent()` method.

Reply:
ok